mlflow/pytorch/__init__.py

```diff
@@ -612,24 +809,25 @@ def predict(self, data, device="cpu"):
             return predicted
 
 
-def autolog(log_every_n_epoch=1):
+def autolog(log_every_n_epoch=1, log_models=True):
```

Comment:
This is an upstream change. Upstream master needs to be synced with forked branch.